docs/dev/test-configuration-options.md

```diff
@@ -11,21 +12,24 @@ export const Highlight = ({children, color}) => ( <span style={{
       padding: '0.2rem',
     }}>{children}</span> );
 
-This page includes a list of valid test configuration options for tests run on Sauce Labs.
+See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the code for setting the capabilities to execute a test.
 
-See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the basic code necessary for executing a test. Those settings as well as all valid parameters are described below.
+Depending on which environment you are running tests in, different options, also known as **Capabilities**, are required. There are different sets of capabilities for different environments, which can be combined. Some of these setting are required for a test to run in a given environment, while some are optional.
+* **[W3C Capabilities:](#browser-w3c-capabilities--optional)** Required for any test using Selenium or Appium to communicate with the browser. W3C capabilities are universal capabilities for any test, and are usually combined with additional capabilities
+* **[Sauce Labs Capabilities:](#desktop-and-mobile-capabilities-sauce-specific--optional)** Needed for running a test on the Sauce Labs Cloud, with different possible sets for different environments. Though there aren't any capabilities required, you will need to [configure the URL](https://wiki.saucelabs.com/pages/viewpage.action?pageId=102704068) and should pass the test name and status as capabilities to the `RemoteWebDriver`.
```

Comment:
I've been trying to avoid programming language specific references. This also doesn't seem to be relevant just to this section? Maybe we can move it up to the top and be more generic: These configurations are added to the Capabilities or Options classes and used to create the Remote WebDriver instance along with the Remote URL that points to a Sauce endpoint.

Reply:
Did a bit of a re-write, will ask for a re-review